slides/src/content/template-projection.md

```diff
@@ -0,0 +1,81 @@
+<!-- .slide: data-background="../content/images/title-slide.jpg" -->
+<!-- .slide: id="template-projection" -->
+## Building Applications with Angular
+
+# Template Projection
+
+---
+<!-- .slide: id="template-projection-roadmap" -->
+## Roadmap
+
+1. How do I project template into my child component?
+1. What options are available?
+
+---
+<!-- .slide: id="template-projection-intro" -->
+## Template Projection
+
+- Ability to pass HTML to a child component and have it rendered there
+  - Best use-case is a component card, to define a header, body and footer
+- This was called _transclusion_ in Angular 1
```

Comment:
This isn't specific to just this card - but in general. But one thing I've noticed last time I did training, and when going to companies for Angular consulting - most of the people have not used Angular 1, and doing comparisons to Angular 1 can end up confusing people.  

I haven't run a full training course in awhile - but wondering if this has been the experience for other people recently?